apps/linode-marketplace-harbor/roles/harbor/templates/motd.j2

```diff
@@ -0,0 +1,7 @@
+*********************************************************
+Akamai Connected Cloud Harbor Marketplace App
+App URL: https://{{ _domain }}
+Credentials File: /root/.credentials
```

Comment:
It doesn't appear that this creds file is generated in the playbook. You can copy what we did for the other apps
https://github.com/akamai-compute-marketplace/marketplace-apps/blob/main/apps/linode-marketplace-joplin/roles/joplin/tasks/main.yml#L66